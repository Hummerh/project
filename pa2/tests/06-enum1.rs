enum Colors {
    Red,
    Blue,
    Green
}

enum Numbers {
    One,
    Two,
    Three
}

enum ABCs { A, B, C }

