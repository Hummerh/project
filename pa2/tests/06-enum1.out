(crate
    (items
        (enum-def
            (id
                (Colors))
            (enum-ctor-defs
                (enum-ctor-def
                    (id
                        (Red)))
                (enum-ctor-def
                    (id
                        (Blue)))
                (enum-ctor-def
                    (id
                        (Green)))))
        (enum-def
            (id
                (Numbers))
            (enum-ctor-defs
                (enum-ctor-def
                    (id
                        (One)))
                (enum-ctor-def
                    (id
                        (Two)))
                (enum-ctor-def
                    (id
                        (Three)))))
        (enum-def
            (id
                (ABCs))
            (enum-ctor-defs
                (enum-ctor-def
                    (id
                        (A)))
                (enum-ctor-def
                    (id
                        (B)))
                (enum-ctor-def
                    (id
                        (C)))))))